### ISSUE TITLE
Ignore duplicates in yara files

### ISSUE THREAD

**Issue by zaqes** [Neo23x0/Loki#215]
Would be great if it could just ignore duplicate yara rules instead of exiting.

**Comment by Neo23x0** [comment]
Yeah, would be great. For now I recommend not to add duplicate rules.
